source/apps/metadata.txt

```diff
@@ -0,0 +1,250 @@
+                  "clusterCount": 1,
+                  "created": "2019-06-23T05:08:35Z",
+                  "links": [
+                    {
+                      "rel": "self",
+                      "href": "https://cloud.mongodb.com/api/atlas/v1.0/groups/5d0171d479328f10cb4f3037"
+                    }
+                  ]
+                }
+              ],
+              "totalCount": 1
+            }
+
+.. _find-app-id:
+
+Find Your App ID
+----------------
+
+Every App has a unique ID. You use App IDs to specify which app you want
+to use or modify.
+
+You can find your Project ID in the App Services UI or use the Admin API
+or App Services CLI to find it programmatically.
+
+.. tabs-realm-admin-interfaces::
+
+   .. tab::
+      :tabid: ui
+
+      .. procedure::
+
+         .. step:: Find Your App in the Atlas UI
+
+            Log into the `MongoDB Atlas UI
+            <https://cloud.mongodb.com?tck=docs_realm>`_ and then click the
+            :guilabel:`App Services` tab.
+
+            Find your App and click on its summary card to open the App.
+
+            .. figure:: /images/view-your-app.png
+               :alt: An overview of all apps associated with an Atlas Project in the UI
+
+         .. step:: Copy Your App ID
+
+            Click the :guilabel:`Copy App ID` button next to the name of your
+            App near the top of the left navigation menu.
+
+            .. note::
+
+                The UI displays your app name in the top left. When you click the
+                copy button, the UI copies your app ID, not your app name, into
+                your clipboard. For example, the UI would display the name
+                "exampleApp" but copy the id "exampleApp-wlkgs".
+
+            .. figure:: /images/copy-your-app-id.png
+               :alt: The location of the copy app id button in the UI
+
+   .. tab::
+      :tabid: api
+
+      To find an App ID, send a request to the :admin-api-endpoint:`List
+      All Apps <operation/adminListApplications>` API endpoint.
+
+      The endpoint returns a list of Apps associated with the Atlas
+      project. Find the App you're interested in and copy its App ID.
+
+      There are two App ID values you may want:
+
+      - The ``_id`` value, which is an internal objectId hex string. Use this
+        value in place of ``{appId}`` in Admin API endpoints.
+
+      - The ``client_app_id``, which is a unique string that includes the App
+        name. Use this value to connect to your App through GraphQL, the Data
+        API, or a Realm SDK.
+
+      .. io-code-block::
+
+         .. input::
+            :language: bash
+
+            curl https://realm.mongodb.com/api/admin/v3.0/groups/{groupId}/apps \
+              --header 'Authorization: Bearer <access_token>'
+
+         .. output::
+            :language: json
+            :emphasize-lines: 3, 4
+
+            [
+              {
+                  "_id": "5997529e46224c6e42gb6dd9",
```

Comment:
super-nit: kinda odd that there's 2 spaces indentation on the lines btwn `[` and `{`, but 4 spaces between `{` and the properties. could you make uniform?